src/Tuura/Boolean/Options.hs

```diff
@@ -1,4 +1,4 @@
-module Tuura.Plato.BoolToConcept.Options (Options(..), getOptions) where
+module Tuura.Boolean.Options (Options(..), getOptions) where
```

Comment:
I'm talking about the module `Tuura.Boolean.Options`. It's not generally useful (noone will ever use it apart from `BoolToConcept/Main`), so it shouldn't live in `Tuura.Boolean.*`. It should be located in the executable file hierarchy.